### PR TITLE
docs/extend/plugins.md: fix links

### PR DESCRIPTION
**Pull request by kolyshkin**
Fix 3 broken links to other docs.

### CODE REVIEW

**Comment by MHBauer** [comment]
Unless things have changed recently, I don't think these changes are correct, and I think what's there now is actually wrong.

From what I recall, the base reference is to inside the docs folder, and the links should not end in `.md`.

Do these links work correctly when `make docs` is run?

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ plugins.
 ## Types of plugins
 
 Plugins extend Docker's functionality.  They come in specific types.  For
-example, a [volume plugin](/extend/plugins_volume.md) might enable Docker
+example, a [volume plugin](/docs/extend/plugins_volume.md) might enable Docker
```
Comment (MHBauer) [inline docs/extend/plugins.md]: should be `/extend/plugins_volume`

File: docs/extend/plugins.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ plugins.
...
 volumes to persist across multiple Docker hosts and a 
-[network plugin](/extend/plugins_network.md) might provide network plumbing
+[network plugin](/docs/extend/plugins_network.md) might provide network plumbing
```
Comment (MHBauer) [inline docs/extend/plugins.md]: `/extend/plugins_network`

File: docs/extend/plugins.md [hunk 3]
```diff
@@ -73,4 +73,4 @@ of the plugin for help. The Docker team may not be able to assist you.
 ## Writing a plugin
 
 If you are interested in writing a plugin for Docker, or seeing how they work
-under the hood, see the [docker plugins reference](/extend/plugin_api).
+under the hood, see the [docker plugins reference](/docs/extend/plugin_api.md).
```
Comment (MHBauer) [inline docs/extend/plugins.md]: `/extend/plugin_api`

**Comment by MHBauer** [comment]
@moxiegirl where are we on that issue/thought of a header on docs files saying "docs links need to work when running 'make docs', and not in github"?

**Comment by kolyshkin** [comment]
My $0.02 is it's probably trivial to teach `make docs` to remove the `docs/` prefix and `.md` suffix. As a result, the links will work on both github and your web site, which is definitely a good thing. I can look at it if you like.

**Comment by kolyshkin** [comment]
These `make docs` changes I propose won't break any existing stuff, but will make it possible to gradually change docs so the links will work on github, too.

**Comment by thaJeztah** [comment]
@MHBauer thanks! I came here for adding those comments, you beat me to it :+1: 

@kolyshkin absolutely, it's being worked on, but those changes in the tooling aren't there yet, so for now, the changes @MHBauer suggested are correct.

**Comment by moxiegirl** [comment]
@MHBauer We have a solution and are in final stages of testing. Happily we also have a dedicated documentation tools person in the form of @SvenDowideit.  So validation of Markdown and documentation as part of PR builds is also not far behind.

File: docs/extend/index.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ weight = 6
 
 Currently, you can extend Docker by adding a plugin. This section contains the following topics:
 
-* [Understand Docker plugins](/extend/plugins.md)
-* [Write a volume plugin](/extend/plugins_volume.md)
-* [Write a network plugin](/extend/plugins_network.md)
-* [Docker plugin API](/extend/plugin_api.md)
+* [Understand Docker plugins](/docs/extend/plugins.md)
+* [Write a volume plugin](/docs/extend/plugins_volume.md)
+* [Write a network plugin](/docs/extend/plugins_network.md)
```
Comment (moxiegirl) [inline docs/extend/index.md]: `/extend/plugins` without the `.md` works in our current static generator.  Unfortunately, the links won't work in GitHub

**Comment by kolyshkin** [comment]
OK, https://github.com/docker/docker/commit/8fee1c2020186ac100b45e64864b94ae3a169ad5 supercedes (and solves) this, so closing. Thanks everyone!

**Comment by thaJeztah** [comment]
Thanks for your contribution @kolyshkin !

**Pull request closed without merging**